packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -443,7 +443,14 @@ export namespace AbstractVSXExtensionComponent {
 
 const downloadFormatter = new Intl.NumberFormat();
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-const downloadCompactFormatter = new Intl.NumberFormat(undefined, { notation: 'compact', compactDisplay: 'short' } as any);
+const downloadCompactFormatter = new Intl.NumberFormat('en-US', { notation: 'compact', compactDisplay: 'short' } as any);
+const averageRatingFormatter = (averageRating: number): number => Math.round(averageRating * 2) / 2;
+const getAverageRatingTitle = (averageRating: number | undefined): string => {
+    if (!averageRating) {
+        return '';
+    }
+    return `Average rating: ${averageRatingFormatter(averageRating)} out of 5`;
```

Comment:
The following should actually be localized. The same is true for other items in the `get tooltip()` if you're up for it :)

```suggestion
    return nls.localize('theia/vsx-extension/averageRating', 'Average rating: {0} out of 5', averageRatingFormatter(averageRating);
```

Reply:
@yiliang114 I also mentioned `get tooltip()`, do you mind updating the code there to add translations? The tooltip is used when hovering over extensions in the view itself:

![Screen Shot 2022-08-03 at 2 17 26 PM](https://user-images.githubusercontent.com/40359487/182680200-201d4006-0dd1-44d4-bbd5-39f4d64b9506.png)

https://github.com/eclipse-theia/theia/blob/5aeef6c0c683b4e91713ab736957e6655b486adc/packages/vsx-registry/src/browser/vsx-extension.tsx#L273-L289

Reply:
@vince-fugnitto I just took care of `get tooltip()`. I have used `nls.localize` to repalce string of `Publisher`, `License` and `Download count` here. 

By the way, as @msujew said below, I have reset the code of `nls.json` and `nls.zh-cn.json`,  Waiting for the code to be incorporated into master and waiting CI will automatically generate the appropriate translation entries. Is it the right thing to do for me?